tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py

```diff
@@ -777,3 +784,160 @@ def forward_xp(self, inputs, xp):
 def test_expand_dims_invalid(xp, shape, axis):
     a = array_utils.create_dummy_ndarray(xp, shape, 'float32')
     return xp.expand_dims(a, axis)
+
+
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize(*(
+    # Single axis
+    chainer.testing.product({
+        'shape': [(2, 2, 2)],
+        'axis': [*range(3)] + [*range(-1, -3, -1)],
```

Comment:
How about including `axis=None` pattern? (same for all configurartions below)

Reply:
Done.